pkg/generate/config/resource.go

```diff
@@ -56,6 +56,9 @@ type ResourceConfig struct {
 	// very little consistency to the APIs that we can use to instruct the code
 	// generator :(
 	UpdateOperation *UpdateOperationConfig `json:"update_operation,omitempty"`
+	// ReconcileConfig describes options for controlling the reconciliation
```

Comment:
nit: In the godoc, the name should be "Reconcile" and not "ReconcileConfig"

Reply:
done.